server/src/test/resources/keys/README

```diff
@@ -0,0 +1 @@
+The string 'password' is the password for all the encrypted keys.
```

Comment:
This is not true for `DESTestKey.pem`. Could just make a note of that here.

Reply:
I'm going to change the key instead :)